setup.py

```diff
@@ -21,7 +21,8 @@
               'bluesky.callbacks'],
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'prettytable', 'cycler',
-                      'numpy', 'matplotlib', 'super_state_machine'],
+                      'numpy', 'matplotlib', 'super_state_machine',
+                      'historydict'],
```

Comment:
Good catch. Updated history -> historydict in new commit. I think everything else in the recipe is already up to date. (Does it have to be the `build` section? I think the `run` section works -- at least that's how it is set up currently.)